tutorials/clustering.md

```diff
@@ -35,26 +42,15 @@ Now, you can tell your access node that currently contains the multinode databas
 data nodes in order to store distributed hypertables:
 
 ```sql
-SELECT add_data_node('dn1',host=>'<address>', if_not_exists=>'true',
-password=>'<remote_password>', bootstrap_user=>'<superuser>',
-bootstrap_password=>'<superuser_password>');
+SELECT add_data_node('dn1', host=>'<data node 1 address>');
 
-SELECT add_data_node('dn2',host=>'<address>', if_not_exists=>'true',
-password=>'<remote_password>', bootstrap_user=>'<superuser>',
-bootstrap_password=>'<superuser_password>');
+SELECT add_data_node('dn2', host=>'<data node 2 address>');
 ```
 
 Let's go through exactly what the `add_data_node` command is doing. The first
```

Comment:
No need to "prepare the user" here, just start describing. Suggest to remove first sentence (yes, it is not part of this patch, but anyway).

Reply:
done